crates/next-custom-transforms/tests/errors/react-server-components/client-graph/root-params/output.stderr

```diff
@@ -1,5 +1,5 @@
-  x You're importing a component that needs "next/root-params". That only works in a Server Component which is not supported in the pages/ directory. Read more: https://nextjs.org/docs/app/building-
-  | your-application/rendering/server-components
+  x You're importing a component that needs "next/root-params". That only works in a Server Component but one of its parents is marked with "use client", so it's a Client Component.
```

Comment:
these snapshots updated because we weren't passing in an app dir to the test fixture. Feels like this is more what we intended to test anyway...